sig-node/CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing on SIG Node
+
+Welcome!
+
+## For Kubernetes Contributions
```

Comment:
A bunch of links are provided here with no context. I think it would be useful to explain why each of these are helpful. A new contributor is not going to know what enhancements are or why they are relevant to SIG Node.

Reply:
The heading above each link explains.